train/train.py

```diff
@@ -1,39 +1,146 @@
+import numpy as np
+import pandas as pd
+import pickle as pkl
+from pathlib import Path
+from collections import defaultdict
+
+from models import get_model_class
 
 
 def train(
         sites,
         models,
-        predictors
+        predictors=None,
+        predictors_paths=None,
+        log_metrics=["pr2", "nmae"],
+        inner_cv=5,
+        n_iter=20,
+        overwrite_existing_models=False
```

Comment:
I suggest changing the default of overwrite to True. Otherwise, training with a different parameter set will raise errors. Users maybe confused.

Reply:
I personally prefer avoiding overwriting models by accident, so what I did is keep the default `False` but add print statements to notify the user when the model is being loaded instead of trained. Let me know what you think.

Reply:
Why I raised this issue is that I find myself likely to re-run the train command using different parameters. The current code will return different error messages for different models selected due to mismatch in predictor set while by default using the saved models. Anything that would avoid these error messages and remind users to change the overwrite_model setting to True would be good.

Reply:
Ah this is a great point. I implemented a small fix which checks arguments saved to file, and asks the user to specify `overwrite_existing_models=True` if the supplied arguments are different than the ones saved to file. What do you think @ylzhouchris ?

Reply:
that sounds perfect.